experiments/aws/parallel-cluster/cpu/experiment/README.md

```diff
@@ -72,8 +72,8 @@ ssh -i ~/.ssh/milroy1-ldrd-east2-ed2.pem ubuntu@ec2-18-117-224-238.us-east-2.com
 Get the topology:
 
 ```bash
-aws ec2 describe-instance-topology --region us-east-2 --filters Name=instance-type,Values=hpc6a.48xlarge --filters Name=tag-key,Values=cluster-tag-value > topology-2.json
-aws ec2 describe-instances --filters "Name=instance-type,Values=hpc6a.48xlarge" --region us-east-2 > instances-2.json
+aws ec2 describe-instance-topology --region us-east-2 --filters Name=instance-type,Values=hpc6a.48xlarge --filters Name=tag-key,Values=cluster-tag-value > topology.json
```

Comment:
I've been keeping the topology "tagged" with the max size. Assuming this is for CPU (256 max):
```suggestion
aws ec2 describe-instance-topology --region us-east-2 --filters Name=instance-type,Values=hpc6a.48xlarge --filters Name=tag-key,Values=cluster-tag-value > topology-256.json
```

Reply:
Done.